packages/account/src/Components/trading-assessment/trading-assessment-new-user.tsx

```diff
@@ -30,17 +48,19 @@ const TradingAssessmentNewUser = ({
                 return { ...accumulator };
             }, {});
         }
-        onSubmit(getCurrentStep() - 1, process_form_values, null, goToNextStep, should_override);
+        onSubmit(getCurrentStep() - 1, process_form_values, goToNextStep, null, should_override);
     };
 
     return (
         <TradingAssessmentForm
-            assessment_questions={assessment_questions}
             form_value={value}
             onSubmit={handleSubmit}
             onCancel={handleCancel}
             setSubSectionIndex={setSubSectionIndex}
             disabled_items={disabled_items}
+            class_name={''}
```

Comment:
cant we make classname optional type in TradingAssessmentForm instead of passing empty classname here?